homeworks/artur.kobyliatsky_Allocene/GamesProject/style/games.css

```diff
@@ -0,0 +1,293 @@
+body {
+  font-family: "Roboto", sans-serif;
+  margin: 0;
+}
+
+
+/*header section*/
+
+.header {
+  position: fixed;
+  width: 100%;
+  top: 0;
+  left: 0;
+  z-index: 2;
+  height: 70px;
+}
+
+.header::before {
+  content: " ";
+  position: absolute;
+  top: 0;
+  left: 0;
+  height: inherit;
+  width: 100%;
+  background-color: #EF4934;
+}
+
+.header__box {
+  display: flex;
+  align-items: center;
+  height: 100%;
+}
+
+.header__box-content {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 90%;
+  margin: 0 auto;
+  z-index: 2;
+}
+
+.header__logo-section {
+  display: flex;
+  align-items: center;
+  z-index: 3;
+}
+
+.header__logo-section img {
+  cursor: pointer;
+}
+
+.header__menu-section {
+  display: flex;
+  align-items: center;
+  z-index: 2;
+}
+
+.header__menu-section a {
+  text-decoration: none;
+  color: #FFF;
+}
+
+.header__menu-section a:active {
+  transition: all .3s;
+  color: #FFF;
+  padding-bottom: 7px;
+  border-bottom: 1px solid #FFF;
+}
+
+.header__menu-section a:hover {
+  transition: all .3s;
+  color: #FFF;
+  padding-bottom: 7px;
+  border-bottom: 1px solid #FFF;
+}
+
+.menu__list {
+  display: flex;
+  z-index: 2;
+}
+
+.menu__list li {
+  list-style: none;
+  margin: 0 0 0 20px;
+}
+
+.menu__list-link {
+  font-size: 16px;
+  text-decoration: none;
+}
+
+
+/*main section*/
+
+main {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.contain {
+  width: 90%;
+  display: grid;
+  grid-template-rows: 1fr 1fr;
+  align-items: center;
+  justify-content: start;
+  margin-top: 180px;
+}
+
+.content {
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  margin-bottom: 100px;
+}
+
+.content .text {
+  font-size: 48px;
+  font-weight: 700;
+  line-height: 32px;
+}
+
+.games__card {
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr 1fr;
+  gap: 25px;
+}
+
+.games__card--cod2 {
```

Comment:
This is a new block, lets say `.card` (what is cod2 means?), not the modifier

Reply:
++ Fixed